src/features/bakery/Bakery.tsx

```diff
@@ -20,6 +21,13 @@ export const Bakery: React.FC = () => {
 
   const isNotReadOnly = !gameState.matches("readonly");
 
+  const kitchenAudio = new Audio(kitchenMP3);
```

Comment:
Do we need to initialise this every time the React component rerenders?

Can we move this inside of the `open` function on line 27?